src/notifications/background/index.js

```diff
@@ -67,7 +67,7 @@ export default class NotificationBackground {
         ))[NotificationBackground.LAST_NOTIF_TIME]
 
         for (let notification of notifications.NOTIFS) {
-            if (notification.system) {
```

Comment:
I realised that this file gets executed once as soon as the user installs it. This leads to all the notifications with the system prop in the object inside the notification.ts file to get called once. I changed it so that only the relevant notification gets called once the user installs.

Reply:
Ah yes, I had a look at the code and I think this was the point of the implementation of that feature. Sure it's a very odd idea, but we were kind of limited with how we could notify users about new things without involving some notification server. 

IIRC the main idea was we just change the stuff in the main static `src/notifications/notifications` module and push out a new update. If the notif in there contains a `system` prop, it means we display it using system notifications (as well as our internal system in the ext overview UI). If we're changing this so that it checks for the release notif (i.e., these changes), we'll have to change this condition every release.

Reply:
This one is still very important to fix up

Reply:
Is it possible to tell me what part of the notification will remain the same? If the id prop remains the same then this issue is easy to fix.

Reply:
Going over this stuff again, and how the whole notifications system works, I actually think we are going about this in the wrong way (apologies that I didn't realize earlier; I'm not entirely familiar with the system).

So (from what I understand) that `NOTIFS` array has the intention of just containing notifs we want to send out on install and update events. They should only ever get sent out once (so on subsequent updates we don't want the same notif being sent from the last update, for example). The 3 notif objects you've added in this work can be considered quite differently, as they definitely don't relate to install and update, rather they get sent out on certain conditions during a backup process. They also could be sent out multiple times if the same conditions are met.

Given those differences, lets keep them separate. Maybe rename that `NOTIFS` array to `UPDATE_NOTIFS` and then create a new one to contain your 3 called `EVENT_NOTIFS`. In here we could keep whatever notifs that fall under that same category - get sent whenever a certain event is triggered. Also it would make more sense to create this new `EVENT_NOTIFS` as an object of ID keys to notif objects, to avoid needing to iterate any array to find a matching notif given an ID (the main use-case would just be "Give me notif with ID XXX").

Does that make sense?

Reply:
Yes. Will fix this and then commit all the changes.